bin/git-fame

```diff
@@ -13,6 +13,7 @@ opts = Trollop::options do
   opt :bytype, "Group line counts by file extension (i.e. .rb, .erb, .yml)", default: false
   opt :include, "Paths to include in git ls-files", default: "", type: String
   opt :exclude, "Paths to exclude (comma separated)", default: "", type: String
+  opt :extension, "Comma-separated list of extensions to consider, leave blank for all", default: "", type: String
```

Comment:
I went with the singular `extension` here, rather than `extensions` based off of my experience building and using HTTP APIs. I'm not sure if that convention also applies to CLI arguments though?